test/test_in_toto_record.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+
+"""
+<Program Name>
+  test_in_toto_record.py
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  Nov 28, 2016
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Test in_toto_record command line tool.
+
+"""
+
+import os
+import sys
+import unittest
+import logging
+import argparse
+from mock import patch
+
+from toto.util import generate_and_write_rsa_keypair
+from toto.models.link import Link
+from toto.in_toto_record import main as in_toto_record_main
+from toto.in_toto_record import in_toto_record_start, in_toto_record_stop
+
+# Suppress all the user feedback that we print using a base logger
+logging.getLogger().setLevel(logging.CRITICAL)
```

Comment:
I don't know if this changes much regarding convenience. In any case I'd like to rethink the logging/user feedback architecture with https://github.com/in-toto/in-toto/issues/6#issuecomment-259750991.